.github/workflows/schema.yml

```diff
@@ -5,15 +5,24 @@ on:
   pull_request:
 
 jobs:
-  generate_schema:
+  generate_and_publish_schema:
     # Deduplicate jobs from pull requests and branch pushes within the same repo.
     if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
     strategy:
       matrix:
         service:
-          - soakdb_sync
-          - pin_packing
-          - targeting
+          - name: compound_library
+            publish: true
+          - name: compound_soaking
+            publish: true
+          - name: crystal_library
+            publish: true
+          - name: soakdb_sync
+            publish: false
+          - name: pin_packing
+            publish: false
+          - name: targeting
+            publish: false
```

Comment:
We want to publish these as they're part of the public interface, even if they don't join up with the existing graph

Reply:
I know, but they but they have conflicts in type definitions, so I thought we keep this PR just for router helm charts and fix those later?

Reply:
Ah yeah, I'd forgotten about that. Sounds sensible